app/views/application/_language.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if I18n.locale == I18n.default_locale %>
+  <%= link_to "It", { locale: "it" } %>
+<% else %>
+  <%= link_to "En", { locale: "en" } %>
+<%end%>
```

Comment:
How about displaying both languages? Thus removing the need to have an `if` block....
In that case, you will want to have an html class to indicate which language is currently selected...